### PR TITLE
chore: implement CLI banner, replace codemod engine commands

### PR DESCRIPTION
**Pull request by DmytroHryshyn**
https://github.com/codemod-com/codemod/assets/125881252/f57e6614-8779-45a6-a387-9972784b121c

https://linear.app/codemod/issue/CDMD-2485/implement-error-banner-ui-that-is-displayed-when
https://linear.app/codemod/issue/CDMD-2486/replace-executable-calls-with-global-cli-calls-remove-code-that-is

### CODE REVIEW

File: apps/vsce/webview/src/CodemodEngineNodeNotFound/index.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { VSCodeTextField } from '@vscode/webview-ui-toolkit/react';
+import cn from 'classnames';
+import styles from './style.module.css';
+
+const INSTALL_CODEMOD_ENGINE_NODE_COMMAND = 'npm install --global codemod';
```
Comment (grzpab) [inline apps/vsce/webview/src/CodemodEngineNodeNotFound/index.tsx:5]: we might shorten it into `npm i -g codemod`

File: apps/vsce/src/components/engineService.ts [hunk 2]
```diff
@@ -228,30 +230,43 @@ export class EngineService {
...
+					kind: MessageKind.codemodEngineNodeLocated,
+				});
+				this.__onCodemodEngineNodeLocated();
+				clearInterval(codemodEnginePollingIntervalId);
+			}
+		};
+
+		checkCodemodEngineNode();
+		// we retry codemod engine installation checks automatically, so we can detect when user installs the codemod
+		const codemodEnginePollingIntervalId = setInterval(
```
Comment (grzpab) [inline apps/vsce/src/components/engineService.ts:253]: when is this interval cancelled?

Comment (DmytroHryshyn) [reply apps/vsce/src/components/engineService.ts:253]: Its cancelled when we successfully locate the engine, in the checkCodemodEngineNode function

Comment (grzpab) [reply apps/vsce/src/components/engineService.ts:253]: Wow, I would assume TS would comment on such an usage of the interval, can we constraint it as well to the max number of checks (e.g. 10?)

**Review by grzpab — APPROVED** [review]

**Pull request merged**